eng/Versions.props

```diff
@@ -40,6 +40,7 @@
     <MicrosoftBclAsyncInterfacesVersion>1.1.0</MicrosoftBclAsyncInterfacesVersion>
     <MicrosoftDiagnosticsRuntimeVersion>2.0.145801</MicrosoftDiagnosticsRuntimeVersion>
     <MicrosoftDiagnosticsRuntimeUtilitiesVersion>2.0.145801</MicrosoftDiagnosticsRuntimeUtilitiesVersion>
+    <ParallelStacksVersion>2.0.1</ParallelStacksVersion>
```

Comment:
I think you want "ParallelStacksRuntimeVersion". This is the reason all CI jobs are failing.

Reply:
This is not that I hate git... I think it is that git hates me...